src/entities/event/ui/event-card/event-card.module.scss

```diff
@@ -78,9 +78,6 @@
 
       .event-additional-info {
         overflow: hidden;
-        display: -webkit-box;
```

Comment:
We would like to limit additional info with 2 lines, so we need `line-clamp: 2`, but now it works only with prefix https://caniuse.com/?search=line-clamp

Reply:
fixed